qiskit_experiments/framework/base_experiment.py

```diff
@@ -265,31 +265,89 @@ def _finalize(self):
         """
         pass
 
-    def _run_jobs(self, circuits: List[QuantumCircuit], **run_options) -> List[Job]:
-        """Run circuits on backend as 1 or more jobs."""
+    def _max_circuits(self, backend: Backend = None):
+        """
+        Calculate the maximum number of circuits per job for the experiment.
+        """
+
+        #set backend
+        if backend is None:
+            if self.backend is None:
+                raise QiskitError("A backend must be provided.")
+            backend = self.backend
+        self.backend = backend
         # Get max circuits for job splitting
-        max_circuits_option = getattr(self.experiment_options, "max_circuits", None)
+        max_circuits_option = getattr(self.experiment_options,
+                                      "max_circuits",
+                                       None)
         max_circuits_backend = self._backend_data.max_circuits
+
         if max_circuits_option and max_circuits_backend:
-            max_circuits = min(max_circuits_option, max_circuits_backend)
+            return min(max_circuits_option, max_circuits_backend)
         elif max_circuits_option:
-            max_circuits = max_circuits_option
+            return max_circuits_option
         else:
-            max_circuits = max_circuits_backend
+            return max_circuits_backend
 
+    def job_info(self, backend: Backend =None):
+        """
+        Get information about job distribution for the experiment on a specific
+        backend.
+
+        Args:
+            backend (Backend): The backend for which to get job distribution 
+            information.
+    
+        Returns:
+            dict: A dictionary containing information about job distribution.
+                - "Total Number of circuits in the experiment": Total number of
+                circuits in the experiment.
+                - "Maximum Number of circuits": Maximum Number of circuits in 
+                one Job
+                - "Total Number of jobs": Number of jobs needed for
```

Comment:
Please update the text here to match the suggested text below. Also, the bulleted list is not rendering correctly, you can check by building the docs locally.